apps/ex_wire/lib/ex_wire/crypto.ex

```diff
@@ -132,4 +132,14 @@ defmodule ExWire.Crypto do
   def node_id_from_public_key(public_key) do
     Key.der_to_raw(public_key)
   end
+
+  @spec hex_to_bin(binary()) :: binary()
+  def hex_to_bin(hex_bin) do
```

Comment:
Want to `import ExthCrypto.Math, only: [hex_to_bin: 1]` like @vyorkin did [here](https://github.com/poanetwork/mana/blob/master/apps/evm/test/evm_test.exs#L2)? I'm good with either as long as we're consistent.